sanowret-crystal.lic

```diff
@@ -0,0 +1,73 @@
+=begin
+	Runs in the background, gazing or exhaling your sanowret crystal. Checks for concentration and current mindstate.
+
+	author: NBSL
+	game: dr
+  version: 0.1 (2017-06-12)
+
+  changelog:
+    0.1 (2017-06-12)
+     * Initial Release
+=end
+
+custom_require.call(%w[common drinfomon])
+
+
+class SanowretCrystal
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'run', regex: /run/i, description: 'Single run of the script', optional: true }
+      ]
+    ]
+    @invalid_room = nil
+    args = parse_args(arg_definitions)
+    check_crystal if args.run
+    passive if args.run.nil?
```

Comment:
Is there any way to check if args exist at all? I have a cleanup script I wrote so that I could train alchemy and not end up with tons of trash and misplaced items, but couldn't figure out how to see if the OpenStruct was empty of args. Thats basically where that if was originally from.